helm/charts/hpe-csi-driver/templates/hpe-csi-node.yaml

```diff
@@ -36,10 +36,6 @@ spec:
             - "--csi-address=$(ADDRESS)"
             - "--kubelet-registration-path=$(DRIVER_REG_SOCK_PATH)"
             - "--v=5"
-          lifecycle:
-            preStop:
-              exec:
-                command: ["/bin/sh", "-c", "rm -rf /registration/csi.hpe.com /registration/csi.hpe.com-reg.sock"]
```

Comment:
What is the downside of not removing this socket?

Reply:
It is a zero-byte file, but the inode won't be reclaimed when the HPE CSI driver isn't used.

Reply:
Should we update the node-driver-registrar image?

https://github.com/kubernetes-csi/node-driver-registrar/pull/61

It seems we are quite dated on ours.

Reply:
Indeed, the latest image does the cleanup automatically.